### ISSUE TITLE
NTR: Cardiac imaging: thick cardiac valves (Lab: ultrasound & echocardiogram)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8700]
New term request Cardiac imaging: thick cardiac valves (elthat_150623022457): Thickening of mitral and aortic valves; echocardiogram; cardiac ultrasound; heart; valvular thickening  UMLS: 3280060

**Comment by pnrobinson** [comment]
@MickeySegal, the HPO would normally have two separate terms for aortic and mitral, 
Thickened aortic valve cusp HP:0031568
there is currently no adequate term for mitral, although this is related --Mitral valve calcification HP:0004382

Can you confirm you mean the cusps rather than the ring, and I will create a new term for mitral analogously? Otherwise please clarify.

**Comment by MickeySegal** [comment]
We have this for a lot of mucopolysaccharidoses.  As an example in https://www.ncbi.nlm.nih.gov/books/NBK1162/ "Progressive thickening and stiffening of the valve leaflets can lead to mitral and aortic regurgitation and stenosis". 

We'd be glad to apply both mitral and aortic terms, or a more general term that includes both.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000342
- label: Thickened mitral valve cusp
- parents: [Abnormal mitral valve leaflet morphology (HP:0031480)](https://hpo.jax.org/app/browse/term/HP:0031480)
- pmid: 
- def: An abnormally increased thickness of a leaflet of the mitral valve.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8702
- synonyms:
